src/lib/cbe/cbe-pool.adb

```diff
@@ -8,6 +8,9 @@
 
 pragma Ada_2012;
 
+with CBE.Debug;
+pragma Unreferenced (CBE.Debug);
```

Comment:
The request pool contains debugging hooks. Therefore, I would like to keep the "with *" and "pragma *". Otherwise, we would have to change code when enabling the debugging.